syntax = "proto2";

package opentxs.proto;
option optimize_for = LITE_RUNTIME;

enum ContactSectionName {
    CONTACTSECTION_ERROR = 0;
    CONTACTSECTION_NAME = 1;
    CONTACTSECTION_EMAIL = 2;
    CONTACTSECTION_PHONE = 3;
    CONTACTSECTION_BITMESSAGE = 4;
    CONTACTSECTION_PROFILE = 5;
    CONTACTSECTION_URL = 6;
    CONTACTSECTION_ACCOMPLISHMENTS = 7;
    CONTACTSECTION_OPENTXS = 8;
    CONTACTSECTION_RELATIONSHIPS = 9;
    CONTACTSECTION_FINGERPRINTS = 10;
}

enum ContactItemType {
    CITEMTYPE_ERROR = 0;
    CITEMTYPE_PERSONAL = 1;
    CITEMTYPE_BUSINESS = 2;
    CITEMTYPE_NICKNAME = 3;
    CITEMTYPE_FACEBOOK = 4;
    CITEMTYPE_TWITTER = 5;
    CITEMTYPE_GOOGLE = 6;
    CITEMTYPE_PROFESSIONAL = 7;
    CITEMTYPE_PARENT = 8;
    CITEMTYPE_CHILD = 9;
    CITEMTYPE_SIBLING = 10;
    CITEMTYPE_EMPLOYER = 11;
    CITEMTYPE_EMPLOYEE = 12;
    CITEMTYPE_MET = 13;
    CITEMTYPE_ALIAS = 14;
    CITEMTYPE_PGP = 15;
    CITEMTYPE_OTR = 16;
    CITEMTYPE_SSL = 17;
}

enum ContactItemAttribute {
    CITEMATTR_ERROR = 0;
    CITEMATTR_ACTIVE = 1;
    CITEMATTR_PRIMARY = 2;
}
